packages/ember/tailwind3-vite/index.js

```diff
@@ -39,6 +44,7 @@ export default async function run() {
         );
 
         contentProperty.value.elements.push(
+          j.literal("index.html"),
```

Comment:
This is IMO always a good idea. Often times one needs to apply `h-screen` to `<html>` or `<body>`.